test/integration/generator.test.js

```diff
@@ -0,0 +1,58 @@
+const { BlobServiceClient } = require('@azure/storage-blob')
```

Comment:
test name needs changed, should reflect the name of the file the call is being made from, i.e. `process-statement-message.test.js`